apps/tudo/src/style/prosemirror.css

```
@@ -0,0 +1,62 @@
+/* Give a remote user a caret */
+.collaboration-carets__caret {
```

Stylelint (17.2.0)

[error] Expected class selector ".collaboration-carets__caret" to be kebab-case (selector-class-pattern)

(selector-class-pattern)